doc/conf/Doxyfile.in

```diff
@@ -47,16 +51,21 @@ GENERATE_BUGLIST       = NO
 GENERATE_DEPRECATEDLIST= NO
 ENABLED_SECTIONS       =
 MAX_INITIALIZER_LINES  = 30
-SHOW_USED_FILES        = YES
+SHOW_USED_FILES        = YES 
 SHOW_FILES             = YES
 SHOW_NAMESPACES        = YES
 FILE_VERSION_FILTER    = "@DIR_SCRIPT@/doxygen_version_extract.sh @PROJECT_SOURCE_DIR@"
-LAYOUT_FILE            =
 CITE_BIB_FILES         = 
+INHERIT_DOCS           = YES
+
+BUILTIN_STL_SUPPORT    = YES
 
-WARN_NO_PARAMDOC       = YES
+WARN_NO_PARAMDOC       = NO 
```

Comment:
I set it to NO because in the dev documentation we get a lot of undocumented function warnings which were annoying. But I did intend to switch it on later. Again, thanks for reminding.

On that note, there are probably a few members which do not have proper documentation. I will try to add something for them.

Reply:
As I see, doxygen complains for only 2 things. First is the [mtx_io](https://github.com/ginkgo-project/ginkgo/blob/develop/core/base/mtx_io.cpp#L67) class which is not documented and the [papi](https://github.com/ginkgo-project/ginkgo/blob/develop/core/log/papi.cpp#L46) which also does not seem to be documented.

Reply:
I think having these warnings should be fine as long as they don't appear on or alter the documentation itself.
And I think `WARN_NO_PARAMDOC` actually warns us when we do not have all parameters documented, which is nice in my opinion, since I do not pay enough attention to these comments when I do code review.